app/src/main/java/com/example/githubusers/model/GithubUsers.java

```diff
@@ -0,0 +1,71 @@
+package com.example.githubusers.model;
+
+import com.google.gson.annotations.SerializedName;
+
+
+public class GithubUsers {
+
+    @SerializedName("username")
```

Comment:
Ensure that the names passed as an argument to the `@SerializedName` notation conform to the fields in the API response. I believe the above is denoted as `login` in the API Response.

Reply:
I did like:
```
@SerializedName("login")
    private String login;```